docs/references/backend/verify-token.mdx

```diff
@@ -25,74 +25,72 @@ function verifyToken: (token: string, options: VerifyTokenOptions) => Promise<Jw
 | `apiUrl?` | `string` | The Clerk [Backend API](/docs/reference/backend-api) endpoint. Defaults to `'https://api.clerk.com'`. |
 | `apiVersion?` | `string` | The version passed to the Clerk API. Defaults to `'v1'`. |
 
 ## Example
 
-The following example demonstrates how to use the [JavaScript Backend SDK](/docs/references/backend/overview) to verify the token signature. Although this example uses Next.js Route Handlers, you can use `verifyToken()` with any JS framework or no framework at all.
+The following example demonstrates how to use the [JavaScript Backend SDK](/docs/references/backend/overview) to verify the token signature.
 
 In the following example:
 
-1. The JWT Public Key is set in the environment variable `CLERK_JWT_KEY`.
+1. The PEM public key from the Clerk Dashboard is set in the environment variable `CLERK_JWT_KEY`.
 1. The session token is retrieved from the `__session` cookie or the Authorization header.
-1. The token is verified using the `verifyToken()` method.
+1. The token is verified in a networkless manner by passing the `jwtKey` to the `verifyToken()` method.
 1. The token's `exp` and `nbf` claims are checked.
 1. The token's `azp` claim is checked against the permitted origins. Verifying the `azp` claim on the server side ensures that the session token is generated from the expected frontend application. For example, if you are permitting tokens retrieved from http://localhost:3000, then the `azp` claim should equal http://localhost:3000.
 1. If the token is valid, the response contains the verified token.
 
-<Tabs items={["Next.js"]}>
-  <Tab>
-  ```ts title="app/api/verify-token/route.ts"
-  import { verifyToken } from '@clerk/backend';
-  import { cookies } from 'next/headers';
+```ts
+import { verifyToken } from '@clerk/backend';
+import { cookies } from 'next/headers';
+
+export async function GET(request: Request) {
+  const cookieStore = cookies()
+  const sessToken = cookieStore.get('__session')?.value
+  const bearerToken = request.headers.get('Authorization')?.replace('Bearer ', '');
+  const token = sessToken || bearerToken;
+
+  if (!token) {
+    return Response.json({ error: 'Token not found. User must sign in.' }, { status: 401 });
+  }
 
-  export async function GET(request: Request) {
-    const cookieStore = cookies()
-    const sessToken = cookieStore.get('__session')?.value
-    const bearerToken = request.headers.get('Authorization')?.replace('Bearer ', '');
-    const token = sessToken || bearerToken;
+  try {
+    const verifiedToken = await verifyToken(token, {
+      jwtKey: process.env.CLERK_JWT_KEY,
+    });
 
-    if (!token) {
-      return Response.json({ error: 'Token not found. User must sign in.' }, { status: 401 });
+    // Check if the token is expired or not yet valid
+    const currentTime = Math.floor(Date.now() / 1000);
+    if (verifiedToken.exp < currentTime || verifiedToken.nbf > currentTime) {
+      throw new Error("Token is expired or not yet valid");
     }
```

Comment:
@alexisintech The above condition should be obsoletesince the `verifyToken()` should validate the `nbf` and `exp` claims. Could you test it and if i am correct, remove the 3 lines above ?